tests/http/clients/async_django.py

```diff
@@ -55,6 +55,13 @@ async def _do_request(self, request: RequestFactory) -> Response:
             return Response(
                 status_code=400, data=e.args[0].encode(), headers=response.headers
             )
+
+        if isinstance(response, StreamingHttpResponse):
```

Comment:
This if/else can be rewritten to avoid some code duplication:

```python
return Response(
    status_code=response.status_code,
    data=(
        response.streaming_content
        if isinstance(response, StreamingHttpResponse) else
        response.content
    ),
    headers=response.headers,
)
```